packages/components/src/services/HomePageService.ts

```diff
@@ -0,0 +1,23 @@
+import { HttpService } from '../utils/httpService';
+import { HomePageInitSchema } from '../schemas';
+
+const apiUrl = 'https://breakup-app-api.herokuapp.com/api';
+
+export const errorMessages = {
+  initCallFailure: 'Failure in init call for fetching data',
```

Comment:
Keep this message in HttpService.ts and use the same message on all network error case

Reply:
descriptive rakna hoga, to track konsi exact call fati
same nhi use krsakte

Reply:
ha, but mostly same hi honge fetch data me, HttpService isme ek failure message ka bana de constant. 

aur wo translation par bhi aage kaam karna h, spanish and hindi side me boht log ne bola h language dalo, review me